src/lsm/manifest_level.zig

```diff
@@ -50,6 +107,11 @@ pub fn ManifestLevelType(
         /// Used to enforce table_count_max_for_level().
         table_count_visible: u32 = 0,
 
+        /// A monotonically increasing generation number that is incremented in insert_table() &
+        /// decremented in remove_table(). This is used detect invalid TableInfo references
+        /// when a caller is using an `internal` TableInfoReference/TableInfoReferenceConst.
```

Comment:
```suggestion
        /// A monotonically increasing generation number that is used detect invalid internal TableInfo references.
```

Don't need to be quite so detailed here imo; "generation" is pretty self-explanatory. Also, the "decremented" part wasn't correct.

Reply:
Oops, my bad, the "decreasing" is indeed wrong; I'll fix this!